app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable, :recoverable,
+  devise :database_authenticatable, :registerable,
+    :rememberable, :validatable
```

Comment:
Minor: Personally I'd just put them in a single line:

```suggestion
  devise :database_authenticatable, :registerable, :rememberable, :validatable
```